<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>446</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>446</width>
    <height>319</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Как вы учились в 5 классе:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>42</y>
      <width>200</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_selected">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>235</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбрано:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вперёд</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Отлично</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хорошо</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Удовлетворительно</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Неудовлетворительно</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
